imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSD.java

```diff
@@ -709,4 +709,5 @@ interface PSD extends com.twelvemonkeys.imageio.metadata.psd.PSD {
     int luni = 'l' << 24 | 'u' << 16 | 'n' << 8 | 'i';
     int lyid = 'l' << 24 | 'y' << 16 | 'i' << 8 | 'd';
     int lsct = 'l' << 24 | 's' << 16 | 'c' << 8 | 't';
+    int lsdk = 'l' << 24 | 's' << 16 | 'd' << 8 | 'k';
```

Comment:
Can you add just a line comment here, like: "// Undocumented: Nested section divider setting" or some similar thing, just so we don't forget what it is here for?